plugins/view-resources/src/components/StringPresenter.svelte

```diff
@@ -15,12 +15,10 @@
 -->
 
 <script lang="ts">
-  import type { Type } from '@anticrm/core'
-
   export let value: string
   
   // eslint-disable-next-line
-  export let attributeType: Type<any>
+  // export let attributeType: Type<any>
```

Comment:
Why we need it?

Reply:
Svelte show warning if presenter is called without property specified, but if it is not used it show error during compile time. So I've not removed since plan to find a way how to solve it for both compile and dev plugin :-)